scipy/spatial/transform/rotation.pyi

```diff
@@ -39,7 +42,7 @@ class Rotation:
     def identity(cls, num: Optional[int] = ...) -> Rotation: ...
     @classmethod
     def random(cls, num: Optional[int] = ...,
-               random_state: Union[int, np.random.RandomState, None] = ...) -> Rotation: ...
+               random_state: Optional[Union[_IntegerType, np.random.Generator]] = ...) -> Rotation: ...
```

Comment:
`RandomState` is still an option, as is `None`.

Reply:
Ah, `Optional` accounts for the `None`. This old-timer is still getting used to the type annotations.